modules/installation-azure-limits.adoc

```diff
@@ -184,6 +184,17 @@ ifndef::ash[]
 endif::ash[]
 |The internal load balancer, each of the three control plane machines, and each
 of the three worker machines each use a private IP address.
+
+ifndef::ash[]
+|Spot VMs
+|
+|20 per region
+|This is an optional component. Spot VM limits are different from the standard VM limits. To use spot VMs, you must increase the spot VM limits.
```

Comment:
I'm confused here. In what way are spot VM limits different? How do you increase spot VM limits? If there's no default limit, why do I need to increase the limit?

Reply:
@MayXuQQ Is there a default limit for spot VMs?

Reply:
I fail to find the default limitation for spot VM, but can control by --max-price when create

Reply:
@ctauchen I have removed the line "spot VMs are different from the standard VMs". And as May Xu pointed, there is no default for how much is required. But, by default Azure offers 20 which is not enough if the users were to use spot VMs.

Reply:
the minimum spot vCPU  20 is OK (3 worker nodes, 3 master nodes, 2 CPUs / worker node, 4 CPUs / master nodes )